calcom/cal.com
# fix: Unable to invite a user to a team via username

## What does this PR do?



- Fixes #15853 (GitHub issue number)
- Fixes CAL-XXXX (N/A)

Old video - `https://www.loom.com/share/3728e0c81bec44a79b5eca54169c6294?sid=6105f4e3-ce2b-4230-bd44-4d6dfccf5b31`

New video -

https://www.loom.com/share/f75dc72f098e4a569bed6bd6bc9cf6c5?sid=1b2d6379-5f05-4717-ab26-d2711f40d2e1

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Create a team.
- Now try to add a member to the team (maybe from here - `http://localhost:3000/settings/teams/<team_id>/members`)
- Instead of writing email, write the username of an existing user.
- It should successfully invite the existing user with username.
